opscenter/6.1/Dockerfile

```diff
@@ -11,6 +11,14 @@ ARG URL_PREFIX=http://localhost
 ARG OPSCENTER_TARBALL=opscenter-${VERSION}.tar.gz
 ARG OPSCENTER_DOWNLOAD_URL=${URL_PREFIX}/${OPSCENTER_TARBALL}
 
+# Set up configuration variables
+ENV OPSCENTER_HOME=/opt/opscenter
+
+RUN set -x \
+# Add Opscenter  user
```

Comment:
I would join this `RUN` statement with the next one, in order to avoid the introduction of a new image layer.